source/_integrations/ruckus_unleashed.markdown

```diff
@@ -16,38 +16,52 @@ ha_platforms:
 ha_integration_type: hub
 ---
 
-This platform allows you to connect to a [Ruckus Unleashed](https://support.ruckuswireless.com/product_families/19-ruckus-unleashed) access point.
+This platform allows you to connect to [Ruckus](https://www.ruckusnetworks.com/) access points.
+
+[Ruckus Unleashed](https://www.ruckusnetworks.com/products/network-control-and-management/controller-less/),
+[Ruckus ZoneDirector](https://support.ruckuswireless.com/products/73),
+[Ruckus SmartZone](https://www.ruckusnetworks.com/products/network-control-and-management/network-controllers/),
+and [Ruckus One](https://www.ruckusnetworks.com/products/network-control-and-management/cloud-managed/)
+access points are supported. Access points running Standalone/Solo firmware are not supported.
 
 There is currently support for the following device types within Home Assistant:
 
-- **Presence detection** - The platform will look at devices connected to the access point and will create a `device_tracker` for each discovered device.
+- **Presence detection** - The platform will look at devices connected to the access point and will
+create a `device_tracker` for each discovered device.
 
 ## Configuration
 
-To add a Ruckus Unleashed device to your installation, go to **Settings** -> **Devices & services**, click the `+` button, then select **Ruckus** from the list of integrations.
+To add a Ruckus Unleashed device to your installation, go to **Settings** -> **Devices & services**,
+click the `+` button, then select **Ruckus** from the list of integrations.
 
-It is required to configure the IP address of your **master access point**. See the section Access Points on the management web interface. And perhaps consider to set a preferred master (Admin & Services>System>System Info>Preferred master).
+Enter be the **IP address / hostname**, **Username** and **Password** which you use to connect
+to your Ruckus controller's web dashboard.
```

Comment:
I can't make a suggestion over deleted lines. But I think this section could be replaced by a prerequisites section, followed by the standard config flow include.
---
## Prerequisites

- **IP address / hostname**, **Username** and **Password** which you use to connect
to your Ruckus controller's web dashboard.

{% include integrations/config_flow.md %}

Reply:
Yes, this is much better thanks!